### PR TITLE
New post: freedom and productivity

### PR DESCRIPTION
**Pull request by bnjbvr**
@adngdb @jankeromnes Can you have a look, please? Thanks!

### CODE REVIEW

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+Title: Freedom and productivity: an open mind manifesto
+Date: 2017-05-12 10:00
+Author: Benjamin Bouvier
+Tags: lifehacking
+Slug: freedom-and-productivity
+Lang: en
+
+![Freedom and productivity]({filename}/images/keep-exploring.jpg)
+
+There are many self-development articles out there on blogs, evocating all the
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: s/evocating/detailing/ ?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+techniques and methods that one can use to work better, be happier, get more
+successful, whatever you can dream of. It really feels these techniques can all
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: feels +like

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+be reduced to a well-defined reduced set of core habits. Work out. Practice
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: -reduced (second one)

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+journaling. Meditate. Be grateful. Read a lot. Drink a liter of water every
+day. Alright, you get the idea; if you're reading this you probably went
+through a whole lot of other very similar self help posts in the past. Or even
+read books about the topic. However, they can easily be misinterpreted. Let's
+find out how. This post is mostly an open-mind manifesto, with some principles
+that might be obvious to some, but for others they might help ease the mind and
+relax.
+
+## When all you've got is a todo-list, everything resembles a task
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Love this title 👍

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+
+Have you ever bought a self-development book and read through the ratings
+section right within the book? It is often placed at the beginning, and it is a
+convincing marketing way to make sure your readers get hooked from the start.
+It gives credibility too; look at all these random people who confirm the
+effectiveness of what you're about to read, before you actually know what
+they're talking about. They expand on how their life changed when they used a
+particular technic, and how they are applying it to every field of their life,
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: s/technic/technique/

Comment (jankeromnes) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: every *aspect* of their life?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+
+I think this is the reason why so many of these books are put aside by most
+people, who materialize an interesting objection: this is the formula for a
+sect-like cult. Is it this stunning that we talk about productivity gurus in
+the first place? Let's think about it for a second: these comments seem to
+suggest that our whole world generates a wide range of different issues that
+can all be addressed with a low number of solutions. You can take any problem
+in your life, turn it in one particular way so as to see it through the lenses
+of this new method, and solve it one way or the other. Every problem can be
+solved with a unique technique. This can be the source of a confirmation bias:
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: s/unique/single/ ?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+if you want it badly enough, and try long enough, you can be pretty sure that
+any issue can fit the scope of that given method you're experimenting with.
+
+If we try to draw a parallel with politics, I'd say these look like the
+premises for a fascist regime (note I am not an historian, so this might be
+incomplete according to my high-school memories). Indeed, some of it is there:
+providing the same answer to all public issues (thus rebranding them, which can
+be facilitated by designating scapegoats), forcing one into adopting behavior
+that might not be suited for them, having a clear distinction between us (who
+have access to The Solution) vs them (who don't/can't have access to it),
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: s/vs/versus/

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+setting up a cult of the personality towards the author of the solution.
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: Interesting point. Personally I don't feel all that connected to the author of a self-help book, I just care about their ideas, but I may not be a representative example.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+
+Why would we act like this? I am no sociologist, and there are probably many
+reasons, but we can explore some basic ideas: maybe *some* people feel
+powerless when facing the complexity of a world where everything is intertwined
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: Not only is everything intertwined in complex ways, but everything around us is also accelerating (a common phrase that may (or may not) help the point).

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+in so many ways. Having a unique, predefined, seemingly theoritically proven
+answer provides them with some sense of order and intellectual easiness. On one
+side, whole ranges of issues then get easily resolved and put aside, which
+gives head space for other matters; on the other side, it means that there are
+no conflicts in thinking, less freedom of choice, no attempts to solutions that
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: Also an interesting point, to which I'm not sure I adhere. There are many self-help books with many ideas, and it seems pretty obvious to me that one shoe does not fit all feet (and that you may have to create a shoe-patchwork by adapting and combining many solutions you've read, sorry for the bad metaphor). Are there really people who read one book and take it as their bible for all The Problems? (I guess so, because I do know examples, but I'm not sure how widespread this problem is).

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 11]
```diff
@@ -0,0 +1,199 @@
...
+are more suited to different groups and contexts, and which ultimately could be
+better overall.
+
+## The fallacies of mimicry
+
+Let's explore another common objection that I call the fallacies of mimicry: as
+if by imitating the actions and habits of so-called successful people, you
+could get their success too. Warren Buffet reads a lot of hours a day; he is
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Random remark: Also Churchill and Thatcher boasted about sleeping only 4h a day, but I've read somewhere (super scientifically) that sleeping less was proven to significantly shrink your lifespan.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 12]
```diff
@@ -0,0 +1,199 @@
...
+insanely rich; so if I read a lot of hours a day, then I'll become rich,
+somehow? I think this kind of reasoning is completely flawed, as it inverts the
+consequences with the premises. Notably, when you get successful enough (which
+seems to translate here by "you don't have to worry about money anymore"), you
+get this great amount of free time that gives you the possibility to read.
+There's another quirk in that thinking: different methods, techniques, habits
+apply to different people in different contexts, and there's no one unique
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Big 👍 on this.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 13]
```diff
@@ -0,0 +1,199 @@
...
+solution that works for all. So repeating the actions of others might not be
+suited for your particular situation, and it could show useless.
+
+That being said, I think there's something else that makes these
+recommendations very appealing; in the end, it's not about reproducing these
+successful stories but much more about having a high-end lifestyle, in which
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: "success stories" might be more idiomatic

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 14]
```diff
@@ -0,0 +1,199 @@
...
+you *take* time for some particular actions and habits, instead of feeling that
+you *lack* time for them. It is much more about being conscious of what really
+matters to you, what you really want to do, rather than getting rich and
+famous. This is not so much about the search of success than the search of
+happiness.
+
+This also raises another point I wonder about: is the seek of productivity only
+a matter of the priviledged? If you consider something like [Maslow's hierarchy
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: s/a matter of/a topic for/

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 15]
```diff
@@ -0,0 +1,199 @@
...
+of needs](https://en.wikipedia.org/wiki/Maslow's_hierarchy_of_needs),
+productivity would probably belong to the top level (self-actualization), since
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: "highest level" might make the sentence even more explicit than "top level" (to me, "top" is too close to "top concern", i.e. what you need to do first; whereas "highest" reminds me more of "high-level concerns" i.e. the opposite of "concrete down-to-earth concerns", sort of).

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+it assumes all other issues in your life have been addressed first. If we put
+aside the working life aspect, a great amount of knowledge can also apply to
+one's personal life and free time. Yet, it feels it's a sign that "success"
+compounds, to some extent: the less you have to worry about universal basic
+needs, the more you can have time to read, learn, get better and enhance
+yourself, which in turn makes you more suited to reach whatever goal you aim
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: Now I really want to see universal basic income arrive to save us all.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 17]
```diff
@@ -0,0 +1,199 @@
...
+for. It doesn't only happen with productivity, but also with education: kids
+from richer families tend to perform better than kids from poorer families, etc
+(there are studies I don't have links to that try to prove this affirmation).
+
+## To seek the whys
+
+There's always something else that's tickling me about those posts: they seem
+to confound the means and the ends. Being productive isn't an end per se, it is
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Gigantic 👍 on "Being productive isn't an end per se". I know it's true, but it seems you can't insist on that enough, as I always get back to my instinctive "I need to work more hours". It's strangely and horribly counter-intuitive.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 18]
```diff
@@ -0,0 +1,199 @@
...
+a tool. And it seems that people forget what is their own motivation in the
+first place, what is their *why*. Why do you want to be more productive or
+successful anyways? Do you want to work less? Or to spend more time with your
+friends and family? Or to spend more time on hobbies and passions that can make
+you feel more fulfilled? Or is your motivation entirely different, and you want
+to raise up in your company's hierarchy, by rising up your throughput, the
+quantity or quality of your work? All motives are valable here of course. But
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: s/or/and/ while we're at it?

Comment (jankeromnes) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: s/valable/valid/ (Cocorico ! 😄 :fr: )

Edit: Ah, maybe you meant "valuable"?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 19]
```diff
@@ -0,0 +1,199 @@
...
+And when the initial, kickstarting motivation is right there, the *why* should
+also help set a limit to the endless quest of productivity. To know when to
+stop. For instance, when you managed to get more time for a dedicated hobby,
+and you're plain satifised, what would be the point of squeezing even more in
+your limited free time? Remember, productivity is a means to an end, not an end
+by itself. Without this in mind, you could be chasing an endless goal.
+
+People get easily fooled into thinking that more working hours imply more
+quality work. That's a societal feeling: in a capitalistic society, being
+productive means contributing to growth thus the good economic health of a
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: s/a entity/an entity/

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 20]
```diff
@@ -0,0 +1,199 @@
...
+entity greater than yourself (your country, your region, the world). And since
+people mix up being productive with working long hours, the confusion is easy
+to make. But I think productivity is the opposite of this: trying to work the
+least number of hours to advance work on the most important and valuable
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: I feel like this could be your title, or your subtitle (very click-baity, seemingly counter-intuitive, but the most valuable point you make to me).

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 21]
```diff
@@ -0,0 +1,199 @@
...
+things. Begin maximally efficient with the minimum amount of resources.
+Identify those actions which will have the most positive impact, help the
+greatest number of people, be *valuable*. So it is another time to acknowledge
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Suggestion: s/it is another time to/this is another opportunity to/ ?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 22]
```diff
@@ -0,0 +1,199 @@
...
+the need for a why: why do you work for, in the first place? Is the work you're
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: -for (why do you work, or what/who do you work for).

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 23]
```diff
@@ -0,0 +1,199 @@
...
+productive? Or more humbly, one might just be working to get the regular pay
+check, since work doesn't have to be a source of fulfillment, but instead can
+just be a means to an end too. Again, all motives are good here; but there
+should be one to make sense out of it.
+
+## To do what you want, to want what you do
+
+I think that's one goal of productivity: to be more conscious of all the
+choices we're making (what to work on) so that our work is more in adequation
+with our principles and real objectives. Thus it is about freedom, that is, the
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: If you have a job that you dislike, just because you need the money and it's hard to see better alternatives, this may hurt a little.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 24]
```diff
@@ -0,0 +1,199 @@
...
+way to express what you truly desire to do and then to do it with purpose. Or
+in the words of French writer and journalist Françoise Giroud: happiness is
+doing what you want, and wanting what you do. I think other people call this
+autonomy or even integrity; I hope you get the idea.
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Great quote!

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 25]
```diff
@@ -0,0 +1,199 @@
...
+
+In particular, I don't want others to preconceive life choices for myself. Even
+if they have theoritized the topic and proven with objective metrics it worked
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: s/it/that it/

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 26]
```diff
@@ -0,0 +1,199 @@
...
+for them. So here is my meta-productivity advice: to try different methods,
+and to make your own solutions out of them, to choose what works according to
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: You could make this sentence bold characters. Just like the one above I said could be your title. They're important and may be the two phrases I'd want to read if I could only read two phrases of your article (which would be quite a shame!).

EDIT: Or highlight it as an out-of-text big side thing (but please not too far from the actual occurence, that's painful).

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 27]
```diff
@@ -0,0 +1,199 @@
...
+first, before applying my own crafted solutions, it is such a lower entry cost
+than doing erratic stuff without purpose and realize I lost my time. Of course,
+I might not find the best solution on the first try, but that's alright. It
+might mean I switch routines from one week to the other, or I don't have a
+consistent agenda; that is okay. I can experiment with many possible ways to do
+one thing, according to my current context: my state of mind, geographical
+location, needs, expectations, levels of energy, unexpected events popping up
+throughout the day, political context, really anything. As long as everything I
+do is conscious and works for me and makes me feel more fulfilled, I'll do the
+tradeoff of not having perfect control and a predefined answer to everything.
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Remark: Sobering/humbling perspective. Well done and thanks!

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 28]
```diff
@@ -0,0 +1,199 @@
...
+
+What some call the review process matters here: one has to make sure they're
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: Readers with no CS background may find "peer review process" more transparent than "review process" (like, the process of putting stars on Amazon products?)

EDIT: Ah, some sort of self-review. I'm not sure about my nit anymore. You can ignore it. 😋

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 29]
```diff
@@ -0,0 +1,199 @@
...
+doing their own right thing, in terms of process and goals, at periodic
+intervals. Otherwise, there's a risk to get drawn in the froth of the days,
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: "froth of the days" ❤️

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 30]
```diff
@@ -0,0 +1,199 @@
...
+into judgement and then feel overwhelmed or unhappy with yourself. You can for
+instance consider you're a scientific researcher that just tries to assess
+hypothesis and confront them with the reality. Or anything else really; as long
+as you keep it fun!
+
+Also, blocking time for rest and non-work activities is important; neuroscience
+has proven that the brain mostly works under two modes, a focused mode (when we
+consciously think about or work on a given problem) and a diffused mode (that
+gets triggered whenever we rest, do errands). Getting stuck in one of the twos
+is the best way to miss better solutions or do subefficient work. It seems
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Amazing advice! I should put these on my wall somewhere.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 31]
```diff
@@ -0,0 +1,199 @@
...
+probably obvious to most that the former is needed, but a lot of people tend to
+underestimate the usefulness of the latter. A lot of background processing
+happens in this mode, and interesting solutions can emerge out of this mind
+state: this might have happened to you through dreams, reveries, showers or
+workout breakthroughs, for instance.
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Great point. I think Stephanie Hurlburt made an amazing tweet about this, let me find it... (20 seconds later) https://twitter.com/sehurlburt/status/891763445586112514 (thank you Google! I never thought "sehurlburt 4 hours a day" would find me this tweet again so easily)

Comment (jankeromnes) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Her second tweet is also amazing: https://twitter.com/sehurlburt/status/891763863091257344

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 32]
```diff
@@ -0,0 +1,199 @@
...
+
+With this in mind, productivity has been much less stressful for me, much more
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: "and much more" ?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 33]
```diff
@@ -0,0 +1,199 @@
...
+of an experimental journey. It has ceased being a tyran always claiming for
```
Comment (jankeromnes) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nit: "tyrant" in English ( 😄 :fr: )

**Review by jankeromnes — APPROVED** [review]
Wow, thanks a lot for writing this excellent and inspiring piece on productivity! I was super happy to read it personally in addition to proof-reading it.

I've added "some" nits. Pro-tip, you can use https://nt1m.github.io/review-comments-viewer/ to resolve them one-by-one (it's what we use as "Reviews" panel in Janitor's IDE, but it actually works stand-alone I can't address nits without it anymore).

Please do send me the link to the post once it's published. I'd like to keep it around. 😄

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 34]
```diff
@@ -0,0 +1,199 @@
...
+Date: 2017-05-12 10:00
+Author: Benjamin Bouvier
+Tags: lifehacking
+Slug: freedom-and-productivity
+Lang: en
+
+![Freedom and productivity]({filename}/images/keep-exploring.jpg)
+
+There are many self-development articles out there on blogs, evocating all the
+techniques and methods that one can use to work better, be happier, get more
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: s/get more/become more

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 4 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Is this a reference to a famous sentence? Because I would write `looks like a task` instead.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 35]
```diff
@@ -0,0 +1,199 @@
...
+people, who materialize an interesting objection: this is the formula for a
+sect-like cult. Is it this stunning that we talk about productivity gurus in
+the first place? Let's think about it for a second: these comments seem to
+suggest that our whole world generates a wide range of different issues that
+can all be addressed with a low number of solutions. You can take any problem
+in your life, turn it in one particular way so as to see it through the lenses
+of this new method, and solve it one way or the other. Every problem can be
+solved with a unique technique. This can be the source of a confirmation bias:
+if you want it badly enough, and try long enough, you can be pretty sure that
+any issue can fit the scope of that given method you're experimenting with.
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: `any issues` (+s)

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 36]
```diff
@@ -0,0 +1,199 @@
...
+
+If we try to draw a parallel with politics, I'd say these look like the
+premises for a fascist regime (note I am not an historian, so this might be
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: s/an historian/a historian

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 37]
```diff
@@ -0,0 +1,199 @@
...
+of needs](https://en.wikipedia.org/wiki/Maslow's_hierarchy_of_needs),
+productivity would probably belong to the top level (self-actualization), since
+it assumes all other issues in your life have been addressed first. If we put
+aside the working life aspect, a great amount of knowledge can also apply to
+one's personal life and free time. Yet, it feels it's a sign that "success"
+compounds, to some extent: the less you have to worry about universal basic
+needs, the more you can have time to read, learn, get better and enhance
+yourself, which in turn makes you more suited to reach whatever goal you aim
+for. It doesn't only happen with productivity, but also with education: kids
+from richer families tend to perform better than kids from poorer families, etc
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: `etc.` (or `…` if you prefer, but `etc` isn't an actual word. :) )

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 17 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: s/confound/confuse? I'm not used to reading the word "confound" but it might just be me.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 38]
```diff
@@ -0,0 +1,199 @@
...
+follow, and thus the practice of productivity routines will be aborted very
+soon after they get started. This is often a very strong sign for myself, when
+I consider doing something, add it to a todo list and months later it is still
+there; maybe it wasn't that useful or valuable, and I can erase it with a light
+heart.
+
+And when the initial, kickstarting motivation is right there, the *why* should
+also help set a limit to the endless quest of productivity. To know when to
+stop. For instance, when you managed to get more time for a dedicated hobby,
+and you're plain satifised, what would be the point of squeezing even more in
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: s/satifised/satisfied

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 39]
```diff
@@ -0,0 +1,199 @@
...
+important to feel like an external observer to yourself, so as to avoid falling
+into judgement and then feel overwhelmed or unhappy with yourself. You can for
+instance consider you're a scientific researcher that just tries to assess
+hypothesis and confront them with the reality. Or anything else really; as long
+as you keep it fun!
+
+Also, blocking time for rest and non-work activities is important; neuroscience
+has proven that the brain mostly works under two modes, a focused mode (when we
+consciously think about or work on a given problem) and a diffused mode (that
+gets triggered whenever we rest, do errands). Getting stuck in one of the twos
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: `the twos`? Les deuxs ? ;D

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 4 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: https://en.wiktionary.org/wiki/if_all_you_have_is_a_hammer,_everything_looks_like_a_nail :)

Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: But I agree it is brilliant!

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 40]
```diff
@@ -0,0 +1,199 @@
...
+![Freedom and productivity]({filename}/images/keep-exploring.jpg)
+
+There are many self-development articles out there on blogs, evocating all the
+techniques and methods that one can use to work better, be happier, get more
+successful, whatever you can dream of. It really feels these techniques can all
+be reduced to a well-defined reduced set of core habits. Work out. Practice
+journaling. Meditate. Be grateful. Read a lot. Drink a liter of water every
+day. Alright, you get the idea; if you're reading this you probably went
+through a whole lot of other very similar self help posts in the past. Or even
+read books about the topic. However, they can easily be misinterpreted. Let's
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: "they" here isn't well defined. I would rephrase this so that it's clearer what it refers to. The previous sentence might be the culprit.

Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Use `those` instead, maybe? Then to me it would be clearer that you refer to books and blog posts.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 8 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: I've seen some examples of writers who are seen as "wildly successful" and thus benefit from an "aura" that borders on guruism. So I tend to agree with Ben on this point. 

I don't much about him, but Seth Godin might be a good example of such a "cult".

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 11 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Yeah, but Churchill was smoking cigars, which I've read somewhere was proven to significantly increase your lifespan. So those might have balanced each others.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 16 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Do you think we can do that with the blockchain?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 20 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Funny because now that I re-read the sentence, out of context, it reads like productivity is the opposite of trying to work the least number of hours. I guess I would rephrase slightly, maybe like so: 

```
But I think productivity is the opposite of this. I think it is trying to work the ...
```

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 41]
```diff
@@ -0,0 +1,199 @@
...
+doing valuable, or are you doing busy work (like handling emails, answering
+very quickly on instant chat systems, etc.) to make it look like you're
+productive? Or more humbly, one might just be working to get the regular pay
+check, since work doesn't have to be a source of fulfillment, but instead can
+just be a means to an end too. Again, all motives are good here; but there
+should be one to make sense out of it.
+
+## To do what you want, to want what you do
+
+I think that's one goal of productivity: to be more conscious of all the
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: s/conscious/aware?

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 23 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Well, as he mentioned earlier in this post, productivity is a topic for rich people anyway... :P

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md [hunk 42]
```diff
@@ -0,0 +1,199 @@
...
+the context. Even if I need to spend a lot of time on thinking things through
+first, before applying my own crafted solutions, it is such a lower entry cost
+than doing erratic stuff without purpose and realize I lost my time. Of course,
+I might not find the best solution on the first try, but that's alright. It
+might mean I switch routines from one week to the other, or I don't have a
+consistent agenda; that is okay. I can experiment with many possible ways to do
+one thing, according to my current context: my state of mind, geographical
+location, needs, expectations, levels of energy, unexpected events popping up
+throughout the day, political context, really anything. As long as everything I
+do is conscious and works for me and makes me feel more fulfilled, I'll do the
```
Comment (adngdb) [inline content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Double `and`. `is conscious, works for me and makes me feel`.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 31 above)
Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Nice tweets! It is comforting to read it, as I have been telling people I work 20 hours per week too. Not that I feel guilty for it, but still, it's nice to hear that I'm not alone. :)

**Review by adngdb — APPROVED** [review]
r+ would review again.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 4 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Thanks, reformulated so it's closer to the original idiom.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 8 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Tim Ferriss and Mr Money Moustache ironically talk about this in one of the podcast the former made, and they're pretty much other examples of self-development heroes.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 10 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Maybe I'm overestimating this issue, but really some review sections in books (like The magic of tidying up, or the Morning Miracle) make you feel you're just being introduced to a sect.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 11 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: [citation needed]

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 17 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: I think `confound` is slightly more appropriate here than `confuse` (a replacement would be `mix up`), so I'll keep it here. It might be more formal language, that being said...

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 18 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Hah, fun: no I really meant `valid`, merci.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 23 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: That's a good point; I'll add a sentence explaining that work is not only what we do in a job, but also activities on our free time.

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 29 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: A poetic display by Boris Vian ("l'écume des jours", which obviously didn't get literally translated like this in English)

File: content/blog/lifehacking/2017-12-01_freedom_and_productivity.md (hunk 36 above)
Comment (bnjbvr) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Amazing. https://www.quora.com/Is-it-an-historian-or-a-historian

Comment (adngdb) [reply content/blog/lifehacking/2017-12-01_freedom_and_productivity.md]: Wow, dingue !
